services/scripts/test-e2e.sh

```
@@ -0,0 +1,253 @@
+        response_time="$(python - <<'PY'
+import sys,decimal
+from decimal import Decimal
+start=Decimal(sys.argv[1]); end=Decimal(sys.argv[2])
+decimal.getcontext().prec=9
+print(end-start)
+PY
+"$start_time" "$end_time")"
+    fi
+
+    # Response time should be less than 2 seconds
+    if python - <<'PY' "$response_time"; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this then clause. Fix to allow more checks.

(SC1073)